release/models/system/openconfig-system-terminal.yang

```diff
@@ -154,6 +154,12 @@ module openconfig-system-terminal {
         "Set the protocol version for SSH connections to the system";
     }
 
+    leaf host-certificate {
+      type string;
+      description
+        "SSH Host Certificate string.";
```

Comment:
Is this the signature or the full content of the host cert?

Should we reference https://datatracker.ietf.org/doc/draft-miller-ssh-cert/ (or other doc of your choice) so there is no ambiguity?

Reply:
SGTM, can you update the description with `This is the full content of an openssh host certificate`